README.md

```
@@ -5,28 +5,32 @@
 [![Coverage Status](https://coveralls.io/repos/github/raeperd/stepdown/badge.svg?branch=main)](https://coveralls.io/github/raeperd/stepdown?branch=main)
 [![Go Reference](https://pkg.go.dev/badge/github.com/raeperd/stepdown.svg)](https://pkg.go.dev/github.com/raeperd/stepdown)
 
-Go linter that checks callers are declared before callees — the **Stepdown Rule** from Clean Code.
+Go linter that keeps your code reading top-to-bottom like a newsletter.
 
-## Why
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)